microsite/src/main/docs/guides/custom-grpc-serialization.md

```diff
@@ -28,21 +28,18 @@ To make the server compress its responses, set the compression type argument to
 
 For example:
 
-```scala mdoc:silent
+```scala
```

Comment:
*question*: is this intentional? I guess we still want to run docs at CI, right?

Reply:
@juanpedromoreno It wasn't really intentional; I had actually _added_ a bunch of `mdoc` tags but several caused failures and I kept getting new errors as I reverted them one at a time. Then I lost patience and just removed them all, including a few previously-existing ones by accident. I just pushed a PR to restore those, hopefully this works.